scripts/README.md

```diff
@@ -48,7 +48,9 @@ Anytime there are _any changes_, you need to run the script that generates the u
 1. Is this the first time that articles have been written for this language? If so:
   * make sure this language appears in the [website config](https://github.com/InnerSourceCommons/innersourcecommons.net/blob/master/config.yaml) and there is an [index page](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/master/content/en/learn/learning-path).
```

Comment:
```suggestion
   * make sure this language appears in the [website config](https://github.com/InnerSourceCommons/innersourcecommons.net/blob/master/config.yaml) and there is an [index page](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/master/content/en/learn/learning-path).
```

Adding more more space to make these bullets sub-lists of the numbered list above.